etc/testing/kube_debug.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -ex
+
+echo "=== TEST FAILED OR TIMED OUT, DUMPING DEBUG INFO ==="
+kubectl get all --all-namespaces
+
+# TODO: Extend this to show kubectl describe output for failed pods, this will
+# probably show why things are hanging.
+
+kubectl version
+kubectl get all --namespace kafka
+kubectl describe pod -l app=pachd
+kubectl describe pod -l suite=pachyderm,app=etcd
+kubectl logs -l app=pachd | tail -n 100
+sudo dmesg |tail -n 20
+top -b -n 1|head -n 20
+df -h
```

Comment:
do these last 3 lines provide helpful output?

Reply:
I suspect they may do in the future, in case we hit OOM, or memory or disk limits.